From b7cd0350fee6fc1fe1306f96635cdc6eb6fe1a0b Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Sat, 20 Aug 2022 12:05:27 +0000
Subject: [PATCH] libstdc++: fix incomplete type of ui::LinuxInputMethodContext

Destructor of std::unique_ptr in libstdc++ uses sizeof() which
requires full definition of ui::LinuxInputMethodContext for return
type of gtk::GtkUiPlatformStub::CreateInputMethodContext().
---
 ui/gtk/gtk_ui_platform_stub.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/ui/gtk/gtk_ui_platform_stub.cc b/ui/gtk/gtk_ui_platform_stub.cc
index 5f01c8b..bfd10bc 100644
--- a/ui/gtk/gtk_ui_platform_stub.cc
+++ b/ui/gtk/gtk_ui_platform_stub.cc
@@ -5,6 +5,7 @@
 #include "ui/gtk/gtk_ui_platform_stub.h"
 
 #include "base/callback.h"
+#include "ui/base/ime/linux/linux_input_method_context.h"
 
 namespace gtk {
 
-- 
2.35.1

